src/command/balance.ts

```diff
@@ -16,24 +17,26 @@ export class Balance extends RootCommand implements LeafCommand {
     await super.init()
 
     this.console.dim('Looking up balance...')
+    const stakeBzz = await this._beeDebug.getStake()
```

Comment:
IMO you can use `beeDebug` instead of `_beeDebug`

The one with underscore prefix does not do a hard requirement for debug api availability, but in this command we need it.